docs/index.md

```diff
@@ -1,154 +1,117 @@
 # Atomist - How Teams Deliver Software
 
+Custom software is crucial to competitive advantage in business. And advantage in custom software
+comes with a custom development experience, including delivery to a unique combination of production environments,
+in your ideal flow.
```

Comment:
This sounds like marketing-speak that would be more at home on the main website.